bot.py

```diff
@@ -19,38 +16,35 @@
 
 # define bot
 intents = discord.Intents().all()
-intents.message_content = True
 client = commands.Bot(command_prefix=PREFIX, intents=intents)
 
+
 # events
 @client.event
 async def on_ready():
     await client.change_presence(activity=Game(name="a CTF!"))
     print(f"Logged in as {client.user.name} ({client.user.id})")
 
+
 @client.event
 async def on_command_error(ctx, error):
     error = getattr(error, 'original', error)
 
-    ignored = (commands.CommandNotFound, )
-    if isinstance(error, ignored):
-        return
-
-    if isinstance(error, commands.DisabledCommand):
-        await ctx.send(f"{ctx.command} has been disabled.")
-
-    elif isinstance(error, commands.NoPrivateMessage):
-        await ctx.author.send(f"{ctx.command} can not be used in Private Messages.")
-
-    elif isinstance(error, commands.MissingRequiredArgument):
-        await ctx.send(f"Missing argument: {error.param}")
-
-    elif isinstance(error, commands.ChannelNotFound):
-        await ctx.send(f"Channel or Category not found: {error.argument}")
+    match error:
```

Comment:
Please revert back to using if / elif. Upgrading to match/case is currently not easily possible.